zomes/hc_zome_trust_atom/src/trust_atom.rs

```diff
@@ -109,14 +117,29 @@ fn trust_atom_link_tag(link_direction: &LinkDirection, mut chunks: Vec<&str>) ->
   let content = chunks.remove(0);
   link_tag_bytes.extend_from_slice(content.as_bytes());
 
-  for chunk in &chunks {
+  for chunk in chunks {
     link_tag_bytes.extend_from_slice(&UNICODE_NUL_BYTES);
     link_tag_bytes.extend_from_slice(chunk.as_bytes());
   }
 
   LinkTag(link_tag_bytes)
 }
 
+fn create_bucket() -> ExternResult<String> {
+  let bucket_bytes = random_bytes(9)?.into_vec();
+  Ok(create_bucket_string(&bucket_bytes))
+}
+
+fn create_bucket_string(bucket_bytes: &[u8]) -> String {
+  let mut bucket = String::new();
+  for chunk in bucket_bytes {
+    let val = chunk;
+    bucket += (val % 10).to_string().as_str();
+    // maybe: ( val.to_float / 25.5 ).to_int
```

Comment:
would be great to do something like this, pretty cheap and makes it feel more pro.  
would want to increase tests a bit more then; like all the boundary conditions:
0, 1, 24, 25, 26, 50, 51, 52, etc up to 255.

but  we could break it out into a separate task if you prefer.

Reply:
Moved to #17